src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -3,13 +3,15 @@
     Choose a base template
 
   %fieldset.templateslist
+    - first = true
     - @projects.each do |project|
       %dl.templateslist
         %dt
           = link_to(title_or_name(project), project_show_path(project))
         - project.packages.each do |template|
           %label
-            %input{ name: 'image', type: 'radio', class: 'hidden' }
+            %input{ name: 'image', type: 'radio', class: 'hidden', checked: first ? true : nil }
```

Comment:
yes! because if `checked: something` is present then the input is checked, doesn't mind the value. So we need it to be `nil`, `false` will make it to be checked too.